src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -4045,29 +4045,29 @@ static LogicalResult verify(ONNXSpaceToDepthOp op) {
   ONNXSpaceToDepthOpAdaptor operandAdaptor(op);
 
   // Check input.
-  auto input = operandAdaptor.input();
+  const Value &input = operandAdaptor.input();
```

Comment:
Should avoid using `auto` when the type is not obvious from the RHS of the assignment. Putting the type explicitly makes the code easier to read as it conveys the static type to the reader immediately (I know modern IDE can display the type ... but some ppl might still want to use bare bone development environments :-))